### PR TITLE
PLAT-129601: Upgrade react-refresh-webpack-plugin 0.3.3 to 0.4.3

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
When update `react-refresh-webpack-pugin` module to 0.4.1, we have "Module not found: Can't resolve 'react-refresh/runtime" error.

### Resolution
Update the module to 0.4.3.

### Links
PLAT-129601

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,6 +132,7 @@
     "fsevents": "^2.1.3"
   },
   "peerDependencies": {
+    "react": ">= 16",
```
Comment (MikyungKim) [inline package.json:135]: Why do we need this?

**Comment by MikyungKim** [comment]
Please update the subject and description of this PR more specifically.

File: package.json (hunk 1 above)
Comment (hong6316) [reply package.json:135]: simply, follow CRA's changes: https://github.com/facebook/create-react-app/pull/9872/files
but without this, 'enact serve -f' works well

Comment (MikyungKim) [reply package.json:135]: The 9872 PR has not landed yet... Why should we take this in advance?

Comment (hong6316) [reply package.json:135]: https://github.com/facebook/create-react-app/pull/9872
: this PR has several changes, and not only for updating plugin ~0.4.3 (add "react" in peerDependencies is not necessary)
i will remove this

**Comment by hong6316** [comment]
> Please update the subject and description of this PR more specifically.

**Review by hong6316 — COMMENTED** [review]
@MikyungKim  please review again

**Review by MikyungKim — APPROVED** [review]
Works fine! :lgtm:

**Pull request merged**